package.json

```diff
@@ -2,17 +2,20 @@
   "name": "contexture-elasticsearch",
   "version": "1.19.7",
   "description": "ElasticSearch Provider for Contexture",
-  "main": "src/index.js",
+  "type": "module",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "files": [
+    "./dist"
+  ],
   "scripts": {
-    "test": "jest",
-    "test:watch": "jest --watch",
```

Comment:
watch mode is helpful

Reply:
I was hoping we could get more acquainted with jest's command line options. Watch mode is just a `yarn test --watch` away. Pretty much the same amount of typing as `yarn test:watch`. Thoughts?